src/protocol/StreamrBinaryMessage.js

```diff
@@ -56,6 +92,17 @@ class StreamrBinaryMessage {
             .Int32BE(contentBuf.length)
         // 20 + streamIdLength: content (variable length)
             .string(contentBuf)
+        if (this.version === VERSION_SIGNED) {
```

Comment:
I wonder if we could have a more object-oriented approach here too? (see the comments in `engine-and-editor` PR)